backstage/templates/backstage-app-config.yaml

```diff
@@ -16,6 +16,10 @@ data:
   {{- if .Values.appConfig.app.googleAnalyticsTrackingId }}
   APP_CONFIG_app_googleAnalyticsTrackingId: {{ .Values.appConfig.app.googleAnalyticsTrackingId | quote | quote }}
   {{- end }}
+  {{- if .Values.appConfig.app.tenant }}
```

Comment:
I would move the value to the root `.Values.tenant` just so that it's the same in both repos.